ledger/metrics/src/main/scala/com/daml/metrics/MetricHandle.scala

```diff
@@ -136,19 +136,10 @@ object MetricHandle {
 
 object MetricDoc {
 
-  sealed trait MetricQualification
-  object MetricQualification {
-    case object Latency extends MetricQualification
-    case object Traffic extends MetricQualification
-    case object Errors extends MetricQualification
-    case object Saturation extends MetricQualification
-    case object Debug extends MetricQualification
-  }
-
   case class Tag(
       summary: String,
       description: String,
-      qualification: MetricQualification,
```

Comment:
I don't understand why it is necessary. What's wrong with `qualification.toString` wherever you need it?